src/dataDisplay/table.jsx

```diff
@@ -77,27 +456,34 @@ Table.Header = TableHeader;
 Table.HeaderCell = TableHeaderCell;
 Table.Row = TableRow;
 
-Table.propTypes = {
-    basic: PropTypes.bool,
-    celled: PropTypes.bool,
-    className: PropTypes.string,
-    collapsing: PropTypes.bool,
-    color: PropTypes.oneOf([ 'bottom', 'middle', 'top' ]),
-    definition: PropTypes.bool,
-    fixed: PropTypes.bool,
-    fontSize: PropTypes.oneOf(Utils.sizeEnums()),
-    fullWidth: PropTypes.bool,
-    id: PropTypes.string,
-    selectable: PropTypes.bool,
-    singleLine: PropTypes.bool,
-    size: PropTypes.oneOf([ 'l', 'large', 'm', 'medium', 's', 'small' ]),
-    stackable: PropTypes.bool,
-    stretch: PropTypes.oneOfType([
-        PropTypes.bool,
-        PropTypes.oneOf([ 'very' ]),
-    ]),
-    striped: PropTypes.bool,
-    style: PropTypes.shape({}),
+Table.propTypes = propTypes;
+Table.defaultProps = defaultProps;
+
+const useStyles = {
```

Comment:
Please move `useStyles` right above the class.

Reply:
done!